src/partr.c

```diff
@@ -130,6 +130,26 @@ void jl_gc_threadfun(void *arg)
         }
         uv_mutex_unlock(&gc_threads_lock);
         gc_mark_loop_parallel(ptls, 0);
+        if (ptls->tid == gc_first_tid) {
```

Comment:
What happens if a mutator thread triggers GC, while page sweeping is concurrently ongoing?

Reply:
We lose that thread for parallel marking.

Reply:
Wondering if we should have a separate "sweep thread" (in additional to the mark threads).